app/views/pages/home.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="container">
+      
+    <% if current_user%>
+          <%= render 'newpost'%>
+    <% end %>
```

Comment:
We don't really need to check for `current_user` here as any unsigned user will be directed to sign in page directly, and won't even reach this page.